.release/docker/talaria_spruce.yaml

```diff
@@ -312,10 +312,10 @@ eventMap:
 jwtValidator:
   Config:
     Resolve:
-      Template: (( grab $THEMIS_ENDPOINT || "http://themis:6500/keys/docker" ))
+      Template: (( grab $THEMIS_ENDPOINT || "http://themis:6500/keys/{keyID}" ))
     Refresh:
       Sources:
-        - URI: (( grab $THEMIS_ENDPOINT || "http://themis:6500/keys/docker" ))
+        - URI: (( grab $THEMIS_ENDPOINT || "http://themis:6500/keys/{keyID}" ))
```

Comment:
Looks like we can remove the `refresh` section since that's what we're doing in our `cd` and `prod` env
- https://github.com/comcast-cl/xmidt/blob/7412b79cb683a8bff664e4b560638ceedbc0022f/rpms/xmidt-na-talaria/talaria.yaml.tp#L77-L800
- https://github.com/comcast-cl/xmidt/blob/7412b79cb683a8bff664e4b560638ceedbc0022f/rpms/xmidt-cd-talaria/talaria.yaml.tp#L77-L80

@johnabass is this because themis doesn't have a `sign-keys/available` api  like sat `https://sat-prod.codebig2.net/v2/sign-keys/available`?